defs/scenarios/storage/ceph/ceph-osd/bugs.yaml

```diff
@@ -52,3 +64,12 @@ conclusions:
         disk. Another potential impact is that it may hit hitting the race condition in Onode::put
         (see https://tracker.ceph.com/issues/56382) which will crash osds, especially in large clusters.
         You can check for the leaked bluestore_cache_other mempool in the output of "ceph daemon osd.id dump_mempools".
+  cephtracker53729:
+    decision:
+      - has_53729
+      - has_53729_osd_log
+    raises:
+      type: CephTrackerBug
+      bug-id: 53729
+      message: >-
+        A really helpful message.
```

Comment:
A suggestion:
"This cluster is vulnerable to the bug https://tracker.ceph.com/issues/53729 in which the OSD(s) consume considerable amount of memory and eventually OOM killed due to potentially millions of duplicate entries for PG(s). Once the bug is hit, it's very hard to recover the OSD(s). So the recommendation is to get the cluster upgraded to a version with the fix at the earliest. Which are 15.2.17 for Octopus (which has a fix to do offline PG trimming via ceph-objectstore-tool), 16.2.11 for Pacific, and 17.2.4 for Quincy.

Ubuntu pakcages to include this fix is tracked via https://bugs.launchpad.net/ubuntu/+source/ceph/+bug/1978913."

We haven't released the full fixes yet for Octopus and Pacific as they're waiting on SRU ^^. Specifically for Octopus, it'd be a update on 15.2.17 after this SRU as 15.2.17 is the last Octopus release (the currently released Octopus only has an offline trimming fix).